content/docs/command-reference/metrics/index.md

```diff
@@ -10,24 +10,37 @@ A set of commands to add, manage, collect, and display project metrics:
 ## Synopsis
 
 ```usage
-usage: dvc metrics [-h] [-q | -v] {show,add,modify,remove} ...
+usage: dvc metrics [-h] [-q | -v] {show,add,modify,remove,diff} ...
 
 positional arguments:
-  COMMAND
-    show                Output metric values.
-    add                 Tag file as a metric file.
-    modify              Modify metric file values.
-    remove              Remove files's metric tag.
+  {show,add,modify,remove,diff}
```

Comment:
Thanks also for updating this one. But actually we use this format specifically in the cmd ref:
```suggestion
COMMAND
```

Reply:
Committing this one for consistency.